cmd/karavictl/cmd/inject.go

```diff
@@ -135,7 +134,74 @@ func NewInjectCmd() *cobra.Command {
 	return injectCmd
 }
 
-func buildProxyContainer(imageAddr, proxyHost string, insecure bool) *corev1.Container {
+func rollout(b []byte, cmd *cobra.Command) {
```

Comment:
Is this function used (commented out above on line 125)? If not it can be removed.